sos/report/plugins/kubernetes.py

```diff
@@ -247,6 +247,24 @@ def postproc(self):
         # output that is not hit by the previous iteration.
         self.do_cmd_private_sub(self.kube_cmd)
 
+        self.do_file_private_sub('/*')
```

Comment:
Absolutely not.

Reply:
This only scrub files copied by this plugin, listing all the RHEL/Ubuntu current paths only open the chance to miss some new paths in the future.
Also if we don't scrub everything, a change in k8s packaging/configuration might make it so that we include a non scrubbed file without any sos changes.

BTW I don't see why `do_file_private_sub` is not done on all files by default, with an option to opt out.

Reply:
now limited to `config_files`